modules/arbeidsgiveravgift/view/salaryreport.php

```diff
@@ -71,7 +71,7 @@
             </tr>
             <tr>
                 <td align="left" colspan="3">Kontorkommune (nummer og navn)</td>
-                <td align="left" colspan="2"><? print $salaryreport->_reportHash['company']['CompanyMunicipality']." ".$salaryreport->_reportHash['company']['CompanyMunicipalityName'] ?><br /></td>
+                <td align="left" colspan="2"><? print $salaryreport->_reportHash['company']['MunicipalityNumber']." ".$salaryreport->_reportHash['company']['MunicipalityName'] ?><br /></td>
```

Comment:
Yeah.
https://github.com/dodoas/lodo/pull/141/files#diff-5a8e6757a15ab299937c605303adda74R50
This is where the value is assigned.